core/API/ResponseBuilder.php

```diff
@@ -164,8 +170,10 @@ private function formatExceptionMessage(Exception $exception)
 
     private function handleDataTable(DataTableInterface $datatable)
     {
-        $postProcessor = new DataTablePostProcessor($this->apiModule, $this->apiMethod, $this->request);
-        $datatable = $postProcessor->process($datatable);
+        if ($this->postProcessDataTable) {
```

Comment:
It is important that the postProcessor cannot be disabled via a request parameter as we would otherwise introduce security issues. Eg people could avoid the execution of the SafeDecodeLabel when PostProcessor is not executed.